porcupine/plugins/directory_tree.py

```diff
@@ -47,8 +47,13 @@
 # where:
 #   - type is "file", "dir", "project"
 #   - project_number is unique to each project
-def get_path(item_id: str) -> Path:
-    item_type, project_number, path = item_id.split(":", maxsplit=2)
+def get_path(item_id: str) -> Path | None:
+    try:
+        item_type, project_number, path = item_id.split(":", maxsplit=2)
+    except ValueError:
+        if "dummy" not in get_directory_tree().item(item_id)["tags"]:
+            raise ValueError
```

Comment:
Nit: It might be best to re-raise the same error. This doesn't really matter though, because if you raise a new erro, the error message will contain both errors.

```suggestion
    except ValueError as e:
        if "dummy" not in get_directory_tree().item(item_id)["tags"]:
            raise e
```

Reply:
Oh I didn't know about this possibility. Thanks!